src/test/java/seedu/address/logic/commands/ReserveCommandTest.java

```diff
@@ -32,14 +31,15 @@
 
     @Test
     public void executeReserve_success() throws CommandException {
-        Booking booking = RESERVE_STUB;
+        /*Booking booking = RESERVE_STUB;
```

Comment:
Are all these test are made by Tuan?

Reply:
No, reserve is by me, but same as above I will work on these tests in a later PR